lib/postal.js

```diff
@@ -72,6 +72,13 @@ ChannelDefinition.prototype.publish = function() {
 	if ( typeof envelope !== "object" ) {
 		throw new Error( "The first argument to ChannelDefinition.publish should be either an envelope object or a string topic." );
 	}
+
+	if (envelope.headers) {
+		envelope.headers.resolverNoCache = _config.resolverNoCache;
+	} else {
+		envelope.headers = {resolverNoCache: _config.resolverNoCache};
+	}
```

Comment:
You could perhaps write this more concisely using `lodash` like so:

```js
envelope.headers = _.assign(envelope.headers || {}, _.pick(_config, ['resolverNoCache']));
```

I haven't tested this, but I think it'll work.